espanol/rebelion/inicio-rebelion.md

```diff
@@ -2,4 +2,6 @@ En un instante el Beatito lo entendió todo, era el momento de iniciar la rebeli
 
 El que estaba "arriba de todos" había ya demostrado ser un tirano, corrompido por el poder y el dinero.
 
-Sin embargo todo se derrumbo al llegar la noche.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.